baseapp-auth/baseapp_auth/rest_framework/users/views.py

```diff
@@ -112,3 +116,31 @@ def permissions(self, request):
         serializer.is_valid(raise_exception=True)
 
         return response.Response({"has_perm": user.has_perm(serializer.data["perm"])})
+
+
+class PermissionsViewSet(
+    NestedViewSetMixin, viewsets.GenericViewSet, mixins.ListModelMixin, mixins.CreateModelMixin
+):
+    pagination_class = None
+    serializer_class = UserManagePermissionSerializer
+    permission_classes = [
+        permissions.IsAuthenticated,
+    ]
+    parent_lookup_kwargs = {"user_pk": "user__id"}
```

Comment:
not sure if this is still true, using `user__id` with 2 `_` will create an SQL with a JOIN and doing `use_id` it will be a simple WHERE instead